### PR TITLE
Change how ID is set on forms, patents, and transparency

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description
- Follow up on conversion off Sandstone
- The nav menu macro needs the body to be set this way instead of with the block
- Also a small formatting fix for the transparency index page.

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/about/policy/patents/guide.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {% extends "mozorg/about-base.html" %}
 
 {% block page_title %}{{ _('Mozilla Patents') }}{% endblock %}
-{% block body_id %}about-patents{% endblock %}
+{% set body_id = "about-patents" %}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/policy/patents/guide.html]: Should this be `about-patents-guide`?

File: bedrock/mozorg/templates/mozorg/about/policy/patents/license-1.0.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 {% extends "mozorg/about-base.html" %}
 
 {% block page_title %}{{ _('Mozilla Open Software Patent License Agreement v1.0') }}{% endblock %}
-{% block body_id %}about-patents{% endblock %}
+{% set body_id = "about-patents-guide" %}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/policy/patents/license-1.0.html]: `about-patents-license-1-0`?

File: bedrock/mozorg/templates/mozorg/about/policy/transparency/includes/past-reports.html [hunk 3]
```diff
@@ -4,7 +4,9 @@
 
 
 {# Add links to previous reports as new reports are published. #}
-<nav class="content nav-previous">
+<div class="nav-previous">
+<div class="content">
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/about/policy/transparency/includes/past-reports.html]: Nit: indentation?

**Comment by stephaniehobson** [comment]
Updated.

**Pull request merged**